content/articles/generating-data-for-a-random-walk-using-python/index.md

```diff
@@ -0,0 +1,145 @@
+In this article, we'll use Python to collect data for a random walk and then use matplotlib to visualize the results. A random walk is a route with no predefined way that is determined by a sequence of random choices made totally at random. A random walk could be compared to the path a cockroach may follow if it lost its mind and takes each walk differently. 
+
+### Prerequisites
+ - Have some basic knowledge of [Python programming language](https://docs.microsoft.com/en-us/windows/python/beginners)
+ - Have [Python](https://www.journaldev.com/30076/install-python-windows-10) and [matplotlib](https://www.tutorialspoint.com/how-to-install-matplotlib-in-python) installed in your machine.
+ - Have some basic understanding of [matplotlib](https://www.simplilearn.com/tutorials/python-tutorial/matplotlib)
+
```

Comment:
Before we get started with the implementation, can you add a sub-topic that discusses where we would use this? Any application of it? How it is used?

Some readers might be new to this.